components/prism-hoon.js

```diff
@@ -0,0 +1,30 @@
+Prism.languages.hoon = {
+	constant: /(?:%\.n)|(?:%\.y)|(?:%[\w-]+\w?)/,
```

Comment:
This can be simplified.
```suggestion
	constant: /%(?:\.[ny])|[\w-]+)/,
```